src/Sentry/Platforms/Native/SentrySdk.cs

```diff
@@ -0,0 +1,60 @@
+using Sentry.Extensibility;
+using Sentry.Native;
+
+// ReSharper disable once CheckNamespace
+namespace Sentry;
+
+public static partial class SentrySdk
+{
+    private static readonly Dictionary<string, bool> PerDirectoryCrashInfo = new();
+
+    private static void InitNativeSdk(SentryOptions options)
+    {
+        if (!C.Init(options))
+        {
+            options.DiagnosticLogger?
+                .LogWarning("Sentry native initialization failed - native crashes are not captured.");
```

Comment:
We should look into if this happens

```suggestion
                .LogError("Sentry native initialization failed - native crashes will not captured. Managed code continues normally.");
```